core/lib/ccm-star.c

```diff
@@ -172,6 +173,7 @@ aead(const uint8_t* nonce,
 /*---------------------------------------------------------------------------*/
 const struct ccm_star_driver ccm_star_driver = {
   set_key,
-  aead
+  aead,
+  mic
```

Comment:
cannot get the mic using aead?
besides, if you update this interfaces, all other implementations of it must also be updated (I'm surprised Travis did not catch this)